packages/edit-site/src/components/page-patterns/dataviews-patterns.js

```diff
@@ -229,7 +241,12 @@ export default function DataviewsPatterns() {
 		],
 		[ view.type, categoryId ]
 	);
-
+	// Reset the page number when the category changes.
+	useEffect( () => {
+		if ( previousCategoryId !== categoryId && view.page !== 1 ) {
+			setView( { ...view, page: 1 } );
```

Comment:
Right now, these categories are built as separate pages, so for me we should reset the entire "view" object when changing categories.

Reply:
I kind of disagree here because while what you say it's true, it feels like we have external filters in the sidebar and for example if I have sorted by title, I'd expect the other categories to be sorted too. No strong opinions for now though, I can update.

Reply:
No strong opinion too. I don't like this effect though and I'm not sure how to avoid it.

Reply:
In my mind this is just a filter that we will need to incorporate in the `view` object. Current patterns page is WIP and has many more things to think about - refactor/simplify etc.. It's just that is there is a lot of complexity and want to tackle things iteratively.

Reply:
This is also close to the "default views" of the pages data views, it's not really clear whether it's filters or just separate default views.

Reply:
> This is also close to the "default views" of the pages data views, it's not really clear whether it's filters or just separate default views.

The "default views" are just presets, preconfigured settings for the view, so we still need a "category" filter in place.